.github/workflows/security-hashicorp-test.yaml

```
@@ -0,0 +1,47 @@
+        description: 'Which branch do you want to use?'
+        required: true
+        default: 'GENIE-948_hashicorp_integration'
+  push:
+    branches:
+      - GENIE-948_hashicorp_integration
+
+jobs:
+  test_hashicorp_vault:
+    runs-on: linux
+    environment:
+      name: ${{ needs.setup_environment.outputs.CLUSTER }}
```

actionlint (1.7.12)

[error] property "setup_environment" is not defined in object type {}

(expression)